pkg/deploy/internal/document/document.go

```diff
@@ -69,7 +68,7 @@ func Deploy(ctx context.Context, client Client, properties parameter.Properties,
 			return createResolvedEntity(documentName, md.ID, c.Coordinate, properties), nil
 		}
 
-		if !isAPIErrorStatusNotFound(err) {
+		if !api.IsNotFoundError(err) {
```

Comment:
On purpose to have both changesets in this PR? :D

Reply:
SonarCloud coverage only triggers if more than 20 lines are changed. This is only a test to check that the coverage data is ingested.

Reply:
20 lines of go production non test code.